cdap-ui/app/cdap/components/DataPrep/DataPrepBrowser/SpannerBrowser/TableList/index.tsx

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {
+  listSpannerTables,
+  listSpannerInstances,
+  listSpannerDatabases,
+  setSpannerLoading,
+  setError,
+} from 'components/DataPrep/DataPrepBrowser/DataPrepBrowserStore/ActionCreator';
+import IconSVG from 'components/IconSVG';
+import {Link} from 'react-router-dom';
+import {getCurrentNamespace} from 'services/NamespaceStore';
+import {objectQuery} from 'services/helpers';
+import LoadingSVGCentered from 'components/LoadingSVGCentered';
+import T from 'i18n-react';
+import MyDataPrepApi from 'api/dataprep';
+
+const PREFIX = `features.DataPrep.DataPrepBrowser.SpannerBrowser`;
+
+interface IReactRouterMatch {
+  params: {
+    connectionId: string;
+    instanceId: string;
+    databaseId: string;
+  };
+}
+
+interface ISpannerTableListViewProps {
+  tableList: object[];
```

Comment:
Same here as well. We should have atleast the `Partial` type of what we get from backend. The properties we use in UI.